src/ui/ResponsiveComponents/ResponsiveTabs.ts

```diff
@@ -344,13 +359,15 @@ export class ResponsiveTabs implements IResponsiveComponent {
   }
 
   private positionPopup() {
-    PopupUtils.positionPopup(
-      this.dropdownContent.el,
-      this.dropdownHeader.el,
-      this.coveoRoot.el,
-      { horizontal: PopupHorizontalAlignment.INNERRIGHT, vertical: PopupVerticalAlignment.BOTTOM },
-      this.coveoRoot.el
-    );
+    this.dropdownContent.insertAfter(this.dropdownHeader.el);
```

Comment:
Should this line be in this method? What happens when `positionPopup` is called more than once? Would that add multiple dropdowns/popups to the header?

To me, this method should just position, whereas now it is acting more like `bindAndPositionPopup`

Also, if I recall correctly, the search-ui usually uses display: none to hide/show dom elements, rather than manipulating the dom using js?

Reply:
It's moving a node around. It's not recreating one.

I prefer to leave it exactly as it was before, the only difference is that I'm adding it after the facet settings so it's next in the document when tabbing.

Before it was appended at the bottom of the document.